daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -1096,6 +1096,14 @@ func (devices *DeviceSet) initDevmapper(doInit bool) error {
 		}
 	}
 
+	if devices.thinPoolDevice == "" {
+		if devices.metadataLoopFile != "" || devices.dataLoopFile != "" {
+			logrus.Errorf("WARNING: LOOP DEVICE FALLBACK: No --storage-opt dm.thinpooldev specified; Usage of loopback is very strongly discouraged for production use.  Use --storage-opt dm.thinpooldev instead.  For more information see `man docker`")
```

Comment:
`logrus.Errorf("WARNING...)` looks little weird. It is error or warning, also you don't need `WARNING` word here.